packages/website/ts/pages/governance/vote_form.tsx

```diff
@@ -193,15 +197,19 @@ export class VoteForm extends React.Component<Props> {
         e.preventDefault();
 
         const { zeip, votePreference, comment } = this.state;
-        const { currentBalance, selectedAddress } = this.props;
+        const { currentBalance, selectedAddress, isLedger } = this.props;
         const makerAddress = selectedAddress;
+
+        if (isLedger) {
```

Comment:
I'm wondering if there is any case where this setState could potentially be ordered after the `setState` that happens in `_handleError`

Reply:
I wasn't too worried about it, but thought of it. Do I have to use a callback to ensure it runs before continuing?